code/modules/modular_computers/computers/item/computer.dm

```diff
@@ -667,12 +667,6 @@
 /obj/item/modular_computer/proc/parent_moved()
 	SEND_SIGNAL(src, COMSIG_MOVABLE_MOVED)
 
-/// Sets visible messages to also send to holder because coders didn't know it didn't do this
-/obj/item/modular_computer/visible_message(message, self_message, blind_message, vision_distance = DEFAULT_MESSAGE_RANGE, list/ignored_mobs, visible_message_flags)
```

Comment:
Is messenger the only time pda uses visible messages?

Reply:
not just messenger i believe

Reply:
if any modular_computer uses visible_message it would be likely to double the messages